components/authn-service/tokens/integration/token_integration_test.go

```diff
@@ -96,11 +115,11 @@ func TestChefClientAuthn(t *testing.T) {
 	serv, err := server.NewServer(ctx, config, authorizationClient)
 	if err != nil {
 		// SKIP these tests if there's no PG_URL given -- and never skip during CI!
-		if pgURLGiven || os.Getenv("CI") == "true" {
+		if pgCfg == nil {
 			t.Fatalf("opening connector: %s", err)
 		} else {
 			t.Logf("opening database: %s", err)
-			t.Logf(testconstants.SkipPGMessageFmt, pgCfg.PGURL)
+			t.Logf("failed to open test database with PG_URL: %q", pgCfg.PGURL)
```

Comment:
These subsequent logs seem confusing-- if we know we're going to log a failure, why log the attempt?

Reply:
I think we are just logging the error and the URL it failed with?

Reply:
Could be more clear though. Open to whatever.

Reply:
We hit this line if the `PG_URL` wasn't provided, right? Can we log that and indicate that we are skipping pg tests (something more akin to the old message)?

Reply:
Basically I would be led astray by the word "failed"

Reply:
I think L119 is where we're handling a nil PG_URL, so maybe we could change the messaging on that one.

Reply:
Yeah same